tutorials/add-disk-configure-datastore-esxi/index.mdx

```diff
@@ -10,17 +10,21 @@ categories:
   - dedibox
 hero: assets/scaleway_add_disk.webp
 dates:
-  validation: 2023-08-22
+  validation: 2024-02-26
   posted: 2020-01-21
 ---
 
-On [Dedibox](https://www.scaleway.com/en/dedibox/), you can order as many [additional disks](https://www.scaleway.com/en/dedibox/storage/) as you need. They can be used as additional Datastore in [ESXi](https://www.vmware.com/products/esxi-and-esx.html) to store your virtual machines, for example.
+With [Dedibox](https://www.scaleway.com/en/dedibox/), you have the flexibility to order [additional disks](https://www.scaleway.com/en/dedibox/storage/) to increase the storage capacity of your machine.
+These disks serve as invaluable resources, particularly when used as additional Datastore within [ESXi](https://www.vmware.com/products/esxi-and-esx.html) setups for storing virtual machines and related data.
 
-A range of different SATA, SAS, and SSD disks is available. Depending on the server chassis, the number of available slots may differ.
+Our offers include a diverse selection of SATA, SAS, and SSD disks, catering to various performance and storage needs.
 
-This guide explains how to order an additional disk from the Dedibox console and how to use it as datastore in a local ESXi environment by following these steps:
+This guide is designed to walk you through the process of ordering an additional disk via the Dedibox console and seamlessly integrating it into your local ESXi environment to function as a Datastore. Follow these steps to make the most of this resource.
 
-Note that instead of adding additional disks to your server, you can also use remote storage based on our RPN-SAN service to add an additional datastore to your ESXi host.
+<Message type="tip">
+  Keep in mind, that the number of available slots for these disks may vary based on the server chassis you choose.
+  Instead of adding additional disks to your server, you can also use remote storage based on our RPN-SAN service to add an additional storage capacity to your ESXi host.
```

Comment:
```suggestion
  Instead of adding additional disks to your server, you can also use remote storage based on our RPN-SAN service to add another storage capacity to your ESXi host.
```